recipe/meta.yaml

```diff
@@ -38,7 +38,9 @@ requirements:
     - pandas
     - prometheus_client
     - prompt_toolkit
-    - python
+    - python >=3.7.0
+    - jupyter_client >=7.0.2
```

Comment:
Jupiter client version 7 switched to async event handling and caused problems with many packages. Spyder chose to stick with version 6 because, I think, it is a larger project and needs more time to be made compatible. 

I think we can create a separate recipe to require version 6 of Jupiter client.